spinta/manifests/xsd/helpers.py

```diff
@@ -112,6 +112,13 @@ def __init__(self, xsd: 'XSDReader', node: _Element = None):
         self.parent_model: XSDModel | None = None
         self.is_root_model: bool | None = None
 
+    def __eq__(self, other: XSDModel) -> bool:
+        if self.external["name"] != other.external["name"]:
```

Comment:
Maybe it's worth adding a type check?
`if not isinstance(other, XSDModel):
        return False
`